### DISCUSSION TITLE
Subdomain routing

### DISCUSSION THREAD

**Discussion by jerry-whoami** [filamentphp/filament#13822] [Help]
### Package

Panel builder

### Package Version

v3

### How can we help you?

I would like to use subdomain routing using filament v3 and nginx

I have my admin panel as

mydomain.com/admin

which works fine

I have a second subdomain called support which is setup as follows:

`return $panel

      ->id('support')

      ->path('/support')

      ->domain('support.mydomain.com')

      ->login(Login::class)

...`

It works kinda.
To access this panel I have to use the url

support.mydomain.com/support

How can I access this panel as

support.mydomain.com  (without the trailing support)

Thanks

**Comment by rvzug** [comment]
See the docs: https://filamentphp.com/docs/3.x/panels/configuration#setting-a-domain

And for using without a path: https://filamentphp.com/docs/3.x/panels/configuration#changing-the-path

**Reply by jerry-whoami** [reply]
Thanks for following up.
I updated my post which was missing the domain call.

So I am setting up the domain as per the docs, however what I am struggling with is
how can to access the support panel using the url

support.mydomain.com  (without the trailing support)

**Reply by rvzug** [reply]
```->path()```

And your routes/web.php may not have a ```Route::get('/')``` and no ```Route::get('')```

**Comment by etcustefan** [comment]
Hello ! I have the same issue but on my '/' path I already have a landing page. I would like to admin.mydomain.com match the panel and mydomain.com match the landing page.

**Reply by fusigabs** [reply]
this is how I do it:
```php
->path('')
->domain('admin.mydomain.com');
```

This should make the panel available at 'admin.mydomain.com' without the need for '/admin'. Your landing page should still be accessible at '/'

**Comment by etcustefan** [comment]
ty
